linker/linker_test.go

```diff
@@ -581,7 +603,10 @@ func TestLinkerValidation(t *testing.T) {
 					  option (foo).buzz = "xyz";
 					}`,
 			},
-			expectedErr: `foo.proto:7:16: message Baz: option (foo).buzz: oneof "bar" already has field "baz" set`,
+			expectedErr:            `foo.proto:7:16: message Baz: option (foo).buzz: oneof "bar" already has field "baz" set`,
+			expectedDiffWithProtoc: true,
+			// TODO: This is a bug of protoc (https://github.com/protocolbuffers/protobuf/issues/9125)
+			//  until v22.0. Difference is expected in the test before it is fixed.
```

Comment:
I don't quite follow the reference to version v22.0. Does this mean the bug exists at least up to v22.0? I think you can just omit it, since there's no certain target version for when it might be fixed.

Reply:
deleted the reference